### DISCUSSION TITLE
side-by-side compare for commited diff

### DISCUSSION THREAD

**Discussion by finallyok** [tpope/vim-fugitive#1727] [Q&A] [answered]
I have a wide screen and always prefer vimdiff style diff for efficience.

it used to be,  Gedit (Gtabedit) a commit-id,  I get a list of changed files, hitting <CR> on a particular file I get vimdiff style change. 
This was still so as of Mar 1, but now all gone for few weeks .    All I get  (say Gedit), is a vertical diff of all the files. diffopt setting has no effect.

Is there any setting I can back to previous behavior? I have to say the previous feature is the deciding reason to use this plugin.  

(I am not talking about :G and dd on a changed file, which still use vimdiff style).

Thanks

**Comment by tpope** [comment, marked as answer]
It sounds like you might be looking for `:set foldmethod=syntax`.

**Reply by finallyok** [reply]
Sweet! Yes, that is the setting changed in 156dbcd. Thanks for the helps.
